toolkit/tools/imagegen/attendedinstaller/uitext/uitext.go

```diff
@@ -156,8 +156,3 @@ const (
 	FinishTitle   = "CBL-Mariner Installation Complete"
 	FinishTextFmt = "Total installation time: %v seconds. Press Enter to restart."
 )
-
-// Common for input validation.
-const (
-	AlphaNumeric = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
```

Comment:
I think there might a place that still references this:
`imagegen/attendedinstaller/views/diskview/manualpartitionwidget/manualpartitionwidget.go:657:111: undefined: uitext.AlphaNumeric`